docs/user/user-settings.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This doc is linked no-where? Or is it auto linked on the sidebar?

Reply:
It's auto linked